bears/java/CheckstyleBear.py

```diff
@@ -67,31 +63,9 @@ def create_arguments(
             - geosoft - The Java style followed by GeoSoft. More info at
               <http://geosoft.no/development/javastyle.html>
         """
-        checkstyle_configs = self.known_configs(checkstyle_configs)
+        if checkstyle_configs in known_checkstyles:
+            checkstyle_configs = self.download_cached_file(
+                known_checkstyles[checkstyle_configs],
+                checkstyle_configs + ".xml")
```

Comment:
not for this commit, but somehow `checkstyle_config` is better because it's no plural, that irritates me everytime...